### DISCUSSION TITLE
How can I create such kinda chart with recharts?

### DISCUSSION THREAD

**Discussion by ziyodbek0300** [recharts/recharts#3661] [General]
![Screenshot 2023-07-18 at 13 59 55](https://github.com/recharts/recharts/assets/57743123/3969e008-01a8-43b7-afd7-1b18323c4087)

Now we have already try several ways. For example we set with <image/> tag but it has not rendered at the first render

**Comment by ckifer** [comment]
No ideas off the top of my head. You'd most likely have to make a custom bar that renders with an image tag and text tag atop it

this might help get you started https://recharts.org/en-US/examples/CustomizedRectangleContent

**Reply by ziyodbek0300** [reply]
Thanks

**Reply by nikolasrieble** [reply]
Let us know if this worked out for you. Also, you could consider contributing and adding this as an example to Storybook. I think this would make for a great contribution. ❤️
